apps/web/src/pages/Auth/SignInPage/SignInPage.tsx

```diff
@@ -49,141 +57,554 @@ const SignInPage = () => {
+                >
+                  <Accordion.Trigger data-h2-font-weight="base(700)">
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "Who do I contact if I have questions about GCKey?",
+                      id: "13Cbkq",
+                      description:
+                        "Sixth of the Frequently Asked Questions for logging in",
+                    })}
+                  </Accordion.Trigger>
+                </Accordion.Header>
+                <Accordion.Content>
+                  <Separator
+                    orientation="horizontal"
+                    decorative
+                    data-h2-background-color="base(gray.lighter)"
+                    data-h2-margin="base(x0.5, 0, x1, 0)"
+                  />
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "If you have questions about GCKey, please contact the GCKey team at:",
+                      id: "gw/WSP",
+                      description:
+                        "Sixth answer of the Frequently Asked Questions for logging in - intro sentence",
+                    })}
+                  </p>
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage: "Canada and the United States",
+                      id: "8NNqcv",
+                      description:
+                        "Sixth answer of the Frequently Asked Questions for logging in - valid area for phone number",
+                    })}
+                  </p>
+                  <p>1-855-438-1102</p>
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage: "Text Telephone (TTY/TDD)",
+                      id: "g1MMTj",
+                      description:
+                        "Sixth answer of the Frequently Asked Questions for logging in - second way to 'contact us'",
+                    })}
+                  </p>
+                  <p>1-855-438-1103</p>
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage: "Outside Canada and the United States",
+                      id: "ConlR5",
+                      description:
+                        "Sixth answer of the Frequently Asked Questions for logging in - valid area for phone number",
+                    })}
+                  </p>
+                  <p>1-800-2318-6290</p>
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "Customer Service Representatives are available to assist you by phone, year round, 24 hours a day, 7 days a week.",
+                      id: "HqQzy1",
+                      description:
+                        "Sixth answer of the Frequently Asked Questions for logging in - final sentence",
+                    })}
+                  </p>
```

Comment:
Can we some margin between each line to match the design like so
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/00f7f20d-5912-4fb3-a137-13dbd8f38a68)